api/routes.py

```diff
@@ -81,6 +81,9 @@ def decorated(*args, **kwargs):
 def allows_auth(f):
     """Decorator function for a controller method that supports both
     authenticated and unauthenticated requests.
+
+    NOTE: This decorator might not be necessary; you can probably call
+    BaseCirculationManagerController.request_patron instead.
```

Comment:
I experimented with marking the controllers that use `load_lane` as `@allow_auth` but I think it's actually better to get rid of the decorator.